templates/flavors/aks/cluster-template.yaml

```diff
@@ -33,6 +33,8 @@ spec:
   location: "${AZURE_LOCATION}"
   sshPublicKey: ${AZURE_SSH_PUBLIC_KEY_B64:=""}
   version: "${KUBERNETES_VERSION}"
+  oidcIssuerProfile:
+    enabled: true
```

Comment:
Do I need to enable this? Running a sanity test locally. BRB

Reply:
ya

Reply:
Tested it locally. I must have `AzureManagedControlPlane.Spec.oidcIssuerProfile.enabled:true` in order to have OIDC ConfigMap created. 
``` 
❯ kg secrets -A -w
NAMESPACE                           NAME                                              TYPE                            DATA   AGE
.
default                             aks-28954-aso-kubeconfig                          Opaque                          1      7m23s
default                             aks-28954-aso-secret                              Opaque                          4      14m
.
default                             aks-6439-aso-kubeconfig                           Opaque                          1      31m
default                             aks-6439-aso-secret                               Opaque                          4      39m
.
```

```
❯ kg configmaps -A
NAMESPACE                           NAME                                                   DATA   AGE
.
default                             aks-6439-aso-oidc-issuer-profile                       1      31m
.
```

Context: `aks-6439` has  `AzureManagedControlPlane.Spec.oidcIssuerProfile.enabled:true`  and `aks-28954` has  `AzureManagedControlPlane.Spec.oidcIssuerProfile` omitted, which at the webhook level gets defaulted to `false`.